src/odemis/acq/stream/_helper.py

```diff
@@ -1448,9 +1450,161 @@ def __init__(self, name, detector, dataflow, emitter, **kwargs):
         emtvas: don't put resolution or scale
         """
         if "acq_type" not in kwargs:
-            kwargs["acq_type"] = model.MD_AT_EM
+            kwargs["acq_type"] = model.MD_AT_EBIC
         super().__init__(name, detector, dataflow, emitter, **kwargs)
 
+
+SCANNER_POS_MD = {model.MD_POS, model.MD_POS_COR, model.MD_PIXEL_SIZE, model.MD_PIXEL_SIZE_COR,
+                  model.MD_SHEAR, model.MD_SHEAR_COR, model.MD_ROTATION, model.MD_ROTATION_COR}
+
+
+class IndependentEBICStream(FastScanningDetector):
+    """
+    A special EBIC stream, typically with a EBIC (current) as a detector and its own scanner.
+    It's physically very similar to the SEM stream, but as we want to select just a region
+    it's in practice similar to the CLSettingStream.
+
+    In live view, the ROI is not applied, but the pixelSize is.
+
+    This stream makes it possible to acquire an image simultaneously with the
+    SED in live view, but as a requirement they would need to align dwell time/resolution.
+    """
+    def __init__(self, name, detector, dataflow, emitter, emt_dataflow, **kwargs):
+        """
+        :param emt_dataflow: a DataFlow that can be used to start/stop the emitter
```

Comment:
The docstring parameters are incomplete (and there is lack of typing in the constructor). It's a bit symptomatic for this class.

Reply:
Indeed. The first 4 parameters are standard parameters for all the Stream classes. They are defined on the base class `Stream`. Repeating them on every of the ~20 classes would be very redundant. I'll add 1 line to mention it though.